### PR TITLE
[rllib] Only run one set of tests unless rllib or tune dirs are changed.

### PR DESCRIPTION
**Pull request by ericl**
## Why are these changes needed?

Most of the tests are unlikely to be affected by core changes. Only run the end to end regression ones on every core build.

@simon-mo @sven1977

### CODE REVIEW

**Review by sven1977 — CHANGES_REQUESTED** [review]
Can we at least run the regression (from tuned_examples/regression_tests) every time? Only the tf2.x one. That one takes only 20min-30min or so. It's the first RLlib matrix entry in travis.yml

File: .travis.yml [hunk 1]
```diff
@@ -171,13 +171,13 @@ matrix:
         - PYTHONWARNINGS=ignore
       install:
         - eval `python $TRAVIS_BUILD_DIR/ci/travis/determine_tests_to_run.py`
-        - if [ $RAY_CI_RLLIB_AFFECTED != "1" ]; then exit; fi
+        - if [ $RAY_CI_RLLIB_FULL_AFFECTED != "1" ]; then exit; fi
```
Comment (sven1977) [inline .travis.yml]: Can we change these two lines back to $RAY_CI_RLLIB_AFFECTED?

File: .travis.yml [hunk 2]
```diff
@@ -171,13 +171,13 @@ matrix:
...
         - ./ci/travis/install-bazel.sh
         - ./ci/travis/install-dependencies.sh
         - export PATH="$HOME/miniconda/bin:$PATH"
         - ./ci/suppress_output ./ci/travis/install-ray.sh
       script:
-        - if [ $RAY_CI_RLLIB_AFFECTED != "1" ]; then exit; fi
+        - if [ $RAY_CI_RLLIB_FULL_AFFECTED != "1" ]; then exit; fi
```
Comment (sven1977) [inline .travis.yml]: Can we change these two lines back to $RAY_CI_RLLIB_AFFECTED?

**Review by sven1977 — APPROVED** [review]

**Comment by ericl** [comment]
Yes, this is always running the 2.x tests each time.

**Pull request merged**